app/models/concerns/authorization_concern.rb

```diff
@@ -142,6 +142,12 @@ def lock!
   class_methods do
     MAX_SAME_CHAR_LIMIT = 4
 
+    def configured_password_length
+      default_min, default_max = EnrollRegistry[:enroll_app].setting(:default_password_length_range).item.split("..").map(&:to_i)
+      return Range.new(default_min, default_max) unless EnrollRegistry.feature_enabled?(:strong_password_length)
```

Comment:
this returns the default_password_lenght unless strong password is enabled, but where does it gets the Length when strong password is enabled ?